front/src/applications/operationalStudies/components/ImportTrainSchedule/ImportTrainScheduleModal.jsx

```diff
@@ -206,15 +209,24 @@ export default function ImportTrainScheduleModal(props) {
   async function launchTrainSchedules(params) {
     try {
```

Comment:
use the same try/catch block, the execution flow is really hard to read

Reply:
No I can't, because if the second request fails (the fetch of the trainschedules), we don't want to display that the pathfindings have failed (which happens right now because of a backend bug)